src/main/java/org/truffleruby/interop/RubyStringToJavaStringWithDefaultNode.java

```diff
@@ -21,12 +21,12 @@
  * the given value if not provided.
  */
 @NodeChild(value = "value", type = RubyNode.class)
-public abstract class NameToJavaStringWithDefaultNode extends RubyNode {
+public abstract class RubyStringToJavaStringWithDefaultNode extends RubyNode {
```

Comment:
Maybe just `ToJavaStringWithDefaultNode` since it uses a `ToJavaStringNode` ?

Reply:
Done.